### ISSUE TITLE
Enemies using ammo with the 'place monster' flag will place a monster -next to you-

### ISSUE THREAD

**Issue by pisskop** [CleverRaven/Cataclysm-DDA#16410]
Whereever or at whomever they fire it, it ends up by you.

Ive seen both friendly and hostile spawns pop up, which I assume is it rolling for difficulty.

<a href="http://imgur.com/EIgDvwT"><img src="http://i.imgur.com/EIgDvwT.png" title="source: imgur.com" /></a>

**Comment by Coolthulhu** [comment]
Place monster flag?

**Comment by pisskop** [comment]
Ammo with the act_on_hit and the use action place_monster

It places a monster by you (or asks you to pick a direction) and then runs a difficulty check if that is specified

**Comment by kevingranade** [comment]
To be clear, I think this is a feature request, not a bug report. I don't
think the place monster iuse was ever intended to create a projectile  that
spawns a monster where the projectile lands.
Can you back up and describe what you're trying to do?

**Comment by pisskop** [comment]
A monster was given the 'gun' special attack and its ammo was set to activate where it landed and use the place_monster use action.  The use action is apparently tied to the PC as all of the spawned critters, hostile or not, show up around the player.

**Comment by Coolthulhu** [comment]
> To be clear, I think this is a feature request, not a bug report.

Yeah, it isn't exactly the original intended use.

I think it could be allowed rather easily, but with limitations. Some iuse functions already use their position argument to determine placement. `place_monster` mostly, but not fully, ignores it - it only takes the z-level component. Changing `p->pos()` to just `pos` (the argument) would make it place the monster near the point of impact.

The problem is, it would still use player skills for the skill checks, because all unowned items are actually used by the player. This could be fixed by having a fake NPC use all unowned items, but it would require testing if all items are fine with that.

**Comment by kevingranade** [comment]
> A monster was given the 'gun' special attack
> and its ammo was set to activate where it
> landed and use the place_monster use
> action.

What monster? I assume one you're adding in a mod, but I'm just guessing.
What are you trying to accomplish? It sounds like you want a monster to
make an attack that results in another monster spawning somewhere. If so I
would recommend adding an attack that does it instead of making it use the
ammunition like you outline.

**Comment by pisskop** [comment]
Well . . .   :v

  - I made a critters that uses the grenadier and elite grenadier's special attacks that allows them to use items.
  - I intended it to be a 'queen' monster, spawning every monster of a faction at interval.
  - The grenadier special attacks only target the player; or more accurately they never use them unless the player is detected and they are hostile to the PC.
  - Thus, I wanted to make ammo that spawned the basic monster, that imp critter in the picture above.
